backpack-lint/src/main/java/net/skyscanner/backpack/lint/check/BpkComponentUsageDetector.kt

```diff
@@ -223,6 +223,16 @@ class BpkComponentUsageDetector : Detector(), SourceCodeScanner, XmlScanner {
       webName = "text",
       componentsToReplace = setOf("android.widget.TextView", "TextView", "androidx.appcompat.widget.AppCompatTextView")
     ),
+    TEXT_FIELD(
+      fullName = "net.skyscanner.backpack.text.BpkTextField",
```

Comment:
I think we need to add this to a new component checklist, it's easy to forget to update the rules.

Reply:
There is a danger check for this :)